test/JDBC/input/ownership-restrictions-from-pg.mix

```diff
@@ -0,0 +1,145 @@
+-- tsql
+CREATE LOGIN r1 WITH password = '123';
+GO
+
+CREATE ROLE role1;
+go
+
+-- psql
+CREATE USER test_user WITH PASSWORD '12345678' inherit;
+go
+
+-- psql user=test_user password=12345678
+DROP ROLE master_role1;
+GO
+
+ALTER ROLE master_role1 RENAME TO master_role2;
+GO
+
+-- Give a bbf role the ability to create other roles, login, nologin and new databases should fail
+ALTER ROLE master_role1 LOGIN;
+GO
+
+ALTER ROLE master_role1 NOLOGIN;
+GO
+
+ALTER ROLE master_role1 CREATEDB;
+GO
+
+ALTER ROLE master_role1 CREATEROLE CREATEDB;
+GO
+
+-- Change bbf role's password should fail
+ALTER ROLE master_role1 PASSWORD '123';
+GO
+
+ALTER ROLE master_role1 WITH PASSWORD '123';
+GO
+
+ALTER ROLE master_role1 WITH PASSWORD NULL;
+GO
+
+-- Make a password valid until a particular period for bbf role should fail
+ALTER ROLE master_role1 VALID UNTIL 'NOV 4 12:00:00 2022 +1';
+GO
+
+ALTER ROLE master_role1 VALID UNTIL 'infinity';
+GO
+
+-- Give a bbf role a non-default setting of the x parameter should fail
+ALTER ROLE master_role1 SET x =false;
+GO
+
+-- Give a bbf role a non-default, database-specific setting of the x parameter should fail
+ALTER ROLE master_role1 IN DATABASE postgres set x=false;
+GO
+
+-- psql
+-- Dropping login from psql port should fail
+DROP ROLE r1;
+GO
+
+ALTER ROLE r1 RENAME TO r2;
+GO
+
+ALTER ROLE r1 CREATEDB;
+GO
+
+-- Create a non babelfish role that is a member of master_guest
+-- and enable dropping
+CREATE ROLE r2 IN ROLE master_guest, tempdb_guest, msdb_guest;
+GO
+
+DROP ROLE r2;
+GO
+
+ALTER ROLE r2 RENAME TO r3;
+GO
+
+SET enable_drop_babelfish_role = true;
+GO
+
+ALTER ROLE r2 RENAME TO r3;
+GO
+
+SET enable_drop_babelfish_role = false;
+GO
+
+SET enable_drop_babelfish_role = true;
+GO
+
+DROP ROLE r3;
+GO
+
+SET enable_drop_babelfish_role = false;
+GO
+
+
+CREATE ROLE r4;
```

Comment:
I think adding file names as prefixes to the names would be better and helpful in debugging the issues later(if any).